### ISSUE TITLE
Entity namespace validation

### ISSUE THREAD

**Issue by ktuite** [getodk/central-backend#680]
We'd like to eventually require use of the entities namespace (`http://www.opendatakit.org/xforms/entities`) in dataset/entity-creating forms.

**Comment by lognaturel** [comment]
I don't think this brings user value so let's close! `pyxform` generates the namespaced attributes and elements as desired.
